public/javascripts/application.js

```diff
@@ -585,12 +585,12 @@ function thermometer() {
           stretch = true
         } else { stretch = false }
 
-        goal_amount = parseFloat(banner_goal_text.replace(/,/g, ''))
-        progress_amount = parseFloat(banner_progress_text.replace(/,/g, ''))
+        goal_amount = parseFloat(banner_goal_text.replace(/[^0-9]/g, ''))
+        progress_amount = parseFloat(banner_progress_text.replace(/[^0-9]/g, ''))
```

Comment:
Does this handle thousand separators?

Reply:
yep! i've tested it as shown for "<p>Members, members, members. We want <span class="goal">ssssssssssssssssssss1,000</span> and we've gotten <span class="progress">900</span>.</p>", progress  bar still shows up.
<img width="916" alt="image" src="https://user-images.githubusercontent.com/60383410/183534470-b1073184-cde2-493d-8158-7a2b787632dd.png">

Reply:
Actually it would potentially cause problems if anyone specified any decimal places such as $9.99 (would show as $999) - is that something that's likely to happen or should it be left alone? Decimal points in some languages such as Spanish are commas, also, and thousand seperators are full-stops, so I'm not sure.

Reply:
Yeah, i18n for this banner will be annoying. But AFAIK we've always implicitly used the en-US (removing the comma), so it's fine to stick with that.

Let's update the regex to keep the decimal point though, since we do support that currently.